pkg/clusterd/disk.go

```diff
@@ -31,8 +31,12 @@ import (
 )
 
 var (
-	logger = capnslog.NewPackageLogger("github.com/rook/rook", "inventory")
-	isRBD  = regexp.MustCompile("^rbd[0-9]+p?[0-9]{0,}$")
+	logger         = capnslog.NewPackageLogger("github.com/rook/rook", "inventory")
+	isRBD          = regexp.MustCompile("^rbd[0-9]+p?[0-9]{0,}$")
+	listAllDevices = "all"
+
+	// We need to allow dm- devices (Kubernetes mountpoints) because dm- devices name are used as names for meta device
+	allowDeviceNamePattern = regexp.MustCompile("dm")
```

Comment:
All the dm devices should start with `dm-`, right? Let's make sure a Abit more specific 

```suggestion
	allowDeviceNamePattern = regexp.MustCompile("dm-")
```

Reply:
Hi Alex, sure make sense